src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,26 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Movies {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+type Props = {
+  moviesCard: Movies[];
+};
+
+export const MoviesList: React.FC<Props> = ({ moviesCard }) => (
+  <>
+    <div className="movies">
```

Comment:
```suggestion
  <div className="movies">
```
фрагмент тут не потрібен. Всередині ж всього лиш один елемент - div